modules/compactor/compactor.go

```diff
@@ -19,7 +20,17 @@ import (
 )
 
 const (
-	waitOnStartup = 90 * time.Second
+	// ringAutoForgetUnhealthyPeriods is how many consecutive timeout periods an unhealthy instance
+	// in the ring will be automatically removed.
+	ringAutoForgetUnhealthyPeriods = 2
+
+	// We use a safe default instead of exposing to config option to the user
+	// in order to simplify the config.
+	ringNumTokens = 512
```

Comment:
I would personally go with a smaller value like 128 or 256. Since most Tempo users are using gossip a smaller value would reduce the chatter between components. I think that getting a super even distribution is less important in this component than others.

No data to back that up. Just a gut feeling.

Reply:
This matches the previous hardcoded value that the compactor was using via [lifecycler config conversion](https://github.com/grafana/tempo/blob/v1.2.1/vendor/github.com/cortexproject/cortex/pkg/compactor/compactor_ring.go#L104).   Not opposed to defaulting to a smaller value, but then wondering if it needs to be configurable.